### PR TITLE
Fix REdit tests

### LINKED ISSUE #358: Fix REdit and REditCommandParser unit tests

### PR DESCRIPTION
**Pull request by clarence-chew**
Resolves #358.

### CODE REVIEW

**Review by MinHeinA — APPROVED** [review]
LGTM!

File: src/test/java/foodwhere/logic/commands/REditCommandTest.java [hunk 1]
```diff
@@ -26,23 +26,25 @@ public class REditCommandTest {
 
     private Model model = new ModelManager(TypicalStalls.getTypicalAddressBook(), new UserPrefs());
 
-    /*
     @Test
     public void execute_allFieldsSpecifiedUnfilteredList_success() {
-        Review editedReview = new ReviewBuilder().build();
+        Review firstReview = model.getFilteredReviewList().get(0);
+
+        // set name and address og new review to match
```
Comment (MinHeinA) [inline src/test/java/foodwhere/logic/commands/REditCommandTest.java:33]: Minor spelling mistake `og`  to change to `of`

**Review by MinHeinA — COMMENTED** [review]
One minor issue

**Comment by Hikoya** [comment]
This PR also resolves #173

**Pull request merged**